backend/compact-connect/lambdas/staff-users/tests/function/test_data_model/test_client.py

```diff
@@ -194,6 +195,29 @@ def test_update_user_permissions_compact_to_board_admin(self):
         # Checking that we're getting the whole object, not just changes
         self.assertFalse({'type', 'userId', 'compact', 'attributes', 'permissions', 'dateOfUpdate'} - resp.keys())
 
+    def test_update_user_permissions_no_change(self):
```

Comment:
To clarify- Is this testing something different from the original issue?

Reply:
Correct. I was just reviewing the overall test coverage in this area and thought we would benefit from including this as well.